ts/main.ts

```diff
@@ -358,5 +375,31 @@ function renderCollectionCards(): void {
     $removeButton.setAttribute('class', 'add-button-search-results');
     $removeButton.textContent = '-';
     $priceAndButton.appendChild($removeButton);
+
+    // EVENT LISTENERS for modal actions
+
+    $removeButton.addEventListener('click', (event) => {
+      $dialog.showModal();
+      event.preventDefault();
```

Comment:
I don't believe this is needed. This is usually only needed inside of `submit` events. If this button is inside a form, then you need to make sure it has `type="button"` otherwise it will try and submit the form, hence why i'm assuming you added this